### DISCUSSION TITLE
Navigation vibration on notifications option

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#679] [Q&A] [answered]
We can include an option to vibrate the device in navigation when notifications happen.

It can be useful when device is in pocket and navigate without visible instructions.

Notifications like turn instructions, nearby poi and favorites, etc. with speech or beep.

<img src="https://github.com/user-attachments/assets/e2a7c5bb-e77b-4941-b904-86ef087cc3f9" width="50%"/>

**Comment by WolfgangZ42** [comment]
> with speech or beep.

I would really appreciate this, especially if I can select a beep (and only a beep) in addition to the vibration - 
see https://github.com/devemux86/cruiser/discussions/570#discussioncomment-10734943

**Comment by 0709wiwiwi** [comment]
Locus has a trill like that. As a cyclist, I don't use. If you ride around on Belgian roads everything vibrates anyway.

**Reply by Maverikk65** [reply]
Always get the goooood vibrationsssssss... ;-)

**Comment by devemux86** [comment]
Available for testing in:
- https://github.com/devemux86/cruiser/discussions/690

 (Settings | Voice guidance | Advanced)

**Comment by devemux86** [comment, marked as answer]
- Navigation vibration on notifications option is available in https://github.com/devemux86/cruiser/discussions/701

**Reply by WolfgangZ42** [reply]
Unfortunately, I have to report that the vibration does not work on my Pixel 8, unlike the "beep(s)". I don't really miss the vibrations, it would merely supplement the indication of turns.

**Reply by devemux86** [reply]
@WolfgangZ42 What do you mean, when?

Perhaps you have disabled the vibration in Android settings?

The power of vibration depends on where we have the phone.
It works every time we have an announcement, in simulation, even at the first message when we start navigation.
